doc/source/index.rst

```diff
@@ -1,13 +1,106 @@
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_forest.png">
+                <span class="home-img-plot-overlay">Forest Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_density.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_density.png">
+                <span class="home-img-plot-overlay">Density Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_energy.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_energy.png">
+                <span class="home-img-plot-overlay">Energy Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_posterior.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_posterior.png">
+                <span class="home-img-plot-overlay">Posterior Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_kde_2d.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_kde_2d.png">
+                <span class="home-img-plot-overlay">KDE 2D Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_forest_ridge.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_forest_ridge.png">
+                <span class="home-img-plot-overlay">Forest Ridge Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_parallel.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_parallel.png">
+                <span class="home-img-plot-overlay">Parallel Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_trace.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_trace.png">
+                <span class="home-img-plot-overlay">Trace Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_dot.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_dot.png">
+                <span class="home-img-plot-overlay">Dot Plot</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_ppc.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_ppc.png">
+                <span class="home-img-plot-overlay">Posterior Predictive Checks</span>
+            </div>
+        </a>
+
+        <a href="examples/plot_autocorr.html">
+            <div class="home-img-plot img-thumbnail">
+                <img src="./_images/mpl_plot_autocorr.png">
+                <span class="home-img-plot-overlay">Autocorrelation Plot</span>
+            </div>
+        </a>
+    </div>
+
 |pypi|
```

Comment:
I would keep the badges right under the title or get rid of them. Their main (only?) goal is that you can get an idea about the state of the library at first glance (CI passing, good coverage, published on pypi, has a doi...), if it is not the first/2nd thing you see I think it looses most of the use.

It is also something very dev/contributor based, so it might be good to keep them on the readme but not on the homepage.